src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -122,9 +134,15 @@
   "Adds in default ca config keys/values, which may be overwritten if a value for
   any of those keys already exists in the ca-data"
   [ca-data]
-  (let [defaults {:allow-subject-alt-names default-allow-subj-alt-names
+  (let [cadir (fs/parent (:cacrl ca-data))
+        defaults {:infra-nodes-path (str cadir "/infra_inventory.txt")
```

Comment:
@justinstoller this (infra-nodes-path) needs to be hard coded and not tied to cadir which comes from ca-data and hence can be changed. PE module will not be aware of this and will always deposit it in a fixed location '/etc/puppetlabs/puppetserver/ca'. Sorry I had pinged you about it early last week when I noticed it then.